source/release-notes/1.8.txt

```diff
@@ -21,9 +21,83 @@ Release Notes for mongosync 1.8
 
 **Upcoming**
 
-New Features:
+Updates to ``mongosync`` ``/progress`` API endpoint
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
We'd also like to make the corresponding updates to the progress documentation page https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/api/progress/

Reply:
Documentation updates are in [DOCSP-41853](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/362)